content/actions/hosting-your-own-runners/using-self-hosted-runners-in-a-workflow.md

```diff
@@ -74,5 +74,5 @@ When routing a job to a self-hosted runner, {% data variables.product.prodname_d
 2. The job is then sent to the first matching runner that is online and idle.
```

Comment:
{% data variables.product.prodname_dotcom %} first searches for an online and enabled runner at the repository level, then at the organization level{% ifversion ghes or ghae %}, then at the enterprise level{% endif %}.
- If we don't find an online and enabled runner at any level, the job is queued to all levels and wait for any runner from any level to come online and pickup the job.
   - If the job remains queued for more than 24 hours, the job will fail.
- If we find an online and enabled runner (preferred runner) at a certain level, the job is then sent to the preferred runner.
    - 60 seconds after sending the job, if the job is not picked up by the preferred runner, we will try to send the same job to all other levels as well.
    - If the job remains queued for more than 24 hours, the job will fail.